apps/condo/domains/meter/schema/MeterReading.js

```diff
@@ -123,6 +130,24 @@ const MeterReading = new GQLListSchema('MeterReading', {
 
             return resolvedData
         },
+        validateInput: async ({ context, resolvedData, existingItem }) => {
+            const newItem = { ...existingItem, ...resolvedData }
+            const meterId = get(newItem, 'meter')
+
+            const meter = await Meter.getOne(context, {
+                id: meterId,
+            })
+
+            const emptyFieldsNames = []
+            for (let i = 1; i <= meter.numberOfTariffs; i++) {
+                if (isNil(get(newItem, `value${i}`))) {
+                    emptyFieldsNames.push(`value${i}`)
+                }
+            }
+            if (emptyFieldsNames.length > 0) {
```

Comment:
You should compare exact numbers. What is meter has 2 tariffs and I've entered all 4?

Reply:
Mmm, we've spoken about partial data. But ok, added checking for extra values.